### ISSUE TITLE
WADM JSON-LD: what goes into //target/format?

### ISSUE THREAD

**Issue by kimpham54** [digitalutsc/islandora_web_annotations#197]
According to https://www.w3.org/TR/annotation-model/#external-web-resources under note, the //target/format property **should** indicate media types, such as from this list http://www.iana.org/assignments/media-types/media-types.xhtml#audio - right now it’s just “image” or “video”. Since we were annotating the presentation derivatives based on known viewers, we could declare the format based on that format (for instance, Basic Image viewer using image/jpeg). However, in the case where a different format is used, this value would need to be changed. Possibly a feature in the future to determine the mimetype of the presentation resource and write that into the format of the json-ld datastream.

**Comment by kstapelfeldt** [comment]
This is definitely a priority, but as the work is less reusable and we are anticipating a CLAW version, we will try to get this right in the CLAW version.

**Comment by Natkeeran** [comment]
Yes, the value we currently have for format property is not complaint with the standard.  This does not impact any user facing functionality.  

Determining the mime type will be different for the current release and CLAW.  

Scope:
* Create a function to determine the mimetype of the datastream.
* Adjust the logic to handle the new format value
